webapp/portlet/src/main/webapp/vue-apps/people-list/components/usercard/PeopleUserCompactCard.vue

```diff
@@ -197,9 +197,6 @@ export default {
     }
   },
   computed: {
-    filteredUserNavigationExtensions() {
```

Comment:
This computed property should not be removed here , no?

Reply:
yes it' should n't